docs/rfcs/0007-export-direction.md

```diff
@@ -0,0 +1,221 @@
+# Direction of Flow component exports
+
+* Date: 2023-03-22
+* Author: Paulin Tpdev (@ptodev)
+* PR:
+* Status: Draft
+
+## Summary
+It may be confusing that component exports can be either the component's input or its output. At the moment Flow doesn't enforce either model. People are free to fix and match them using their best judgement. This RFC evaluates the pros and cons of each approach, and also states if it'd be worth it to change our current approach.
```

Comment:
Could we please define what 'inputs' and 'outputs' are early on and keep the terminology consistent throughout the doc? Do we mean _data_ flow? How does this impact components that do not flow data through (eg. an authentication component).